client/post-editor/editor-feedback-invitation/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { identity } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+function EditorFeedbackInvitation( { translate, onTrigger } ) {
+	return (
+		<div className="editor-feedback-invitation">
+			<p className="editor-feedback-invitation__invitation">
+				{ translate(
+					'{{highlight}}Need another set of eyes?{{/highlight}} Ask a friend to review your post.',
+					{
+						components: {
+							highlight: <span className="editor-feedback-invitation__highlight" />
+						}
+					}
+				) }
+			</p>
+			<Button className="editor-feedback-invitation__button" onClick={ onTrigger }>
+				{ translate( 'Send to a Friend' ) }
+			</Button>
+		</div>
+	);
+}
```

Comment:
I like the arrow version in your example but prefer creating named functions where possible. It's probably not a big deal in practice, especially for disciplined React code, but after debugging a lot of code with many layers of indirection and anonymous functions, it makes me feel good to create a function that will have a name in the debugger's stack UI. :)

Thanks for the solid suggestion of exporting the unwrapped component to facilitate testing. I'll update that now.